src/com/facebook/buck/ide/intellij/templates/ij-module.st

```diff
@@ -45,8 +45,12 @@
     %if(androidFacet.compiler_output_path)%inherit-compiler-output="false"%else%inherit-compiler-output="true"%endif%>
 %endif%
     %if(androidFacet.compiler_output_path)%
+    %if(androidFacet.is_compiler_output_path_jar)%
+    <output url="jar://$MODULE_DIR$%androidFacet.compiler_output_path%!/" />
```

Comment:
How about instead of increasing complexity of the template move URL construction to java code and pass one parameter here:

```
    <output url="%androidFacet.compiler_output_path%" />
```